packages/neon_framework/lib/src/utils/request_manager.dart

```diff
@@ -92,13 +92,13 @@ class RequestManager {
         },
         unwrap: (rawResponse) => unwrap(rawResponse),
         serialize: (data) {
-          const encoder = RawResponseEncoder();
-
-          return json.encode(encoder.convert(data as DynamiteRawResponse));
+          return const RawResponseEncoder()
+              .fuse(const JsonEncoder())
+              .fuse(const Utf8Encoder())
```

Comment:
Any reason you did not use the mentioned `JsonUtf8Encoder`?

Reply:
Only to make it more similar to the deserialization part. If you want I can change it, but I don't know if it makes any difference in the end.

Reply:
Ok from the class description it sounds like it needs one less copy of the data and thus should be more performant.